health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/UserAccess.js

```
@@ -3,12 +3,40 @@ import React, { Fragment, useEffect, useState } from "react";
 
+const Wrapper = ({ setShowPopUp, alreadyQueuedSelectedState }) => {
+  const { t } = useTranslation();
+  return (
+    <PopUp
+      className={""}
+      style={{
+        maxWidth: "40%",
+      }}
+      type={"default"}
+      heading={t("MICROPLAN_ADMINISTRATIVE_AREA")}
+      children={[]}
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)